champions/src/main/java/me/mykindos/betterpvp/champions/listeners/ArrowListener.java

```diff
@@ -99,7 +106,7 @@ public void onArrowHit(ProjectileHitEvent event) {
             UtilServer.runTaskLater(champions, () -> {
                 arrow.remove();
                 arrows.remove(arrow);
-            }, 5L);
+            }, 40L);
```

Comment:
This might cause players to be able to pick arrows up if they spam it, can you confirm this?

Reply:
The two reasons behind this change are: 
- Allow enemies to pick up missed arrows by the ranger
- Allow the ranger to better gauge where their arrow lands so they can aim the next shot more accurately

Reply:
This will have to be taken into account for in Champions but fine for now.